src/components/User/Sales/SaleRequests/components/SaleRequestManager/SaleRequestManager.vue

```diff
@@ -211,6 +216,40 @@ export default {
       this.isSubmitting = false
     },
 
+    async makePairTradeable () {
+      try {
+        const { data: pairs } = await ApiCallerFactory
+          .createCallerInstance()
+          .get('/v3/asset_pairs', {
+            filter: { base_asset: this.request.token.code }
+          })
+
+        pairs.forEach(async item => {
+          if (!(item.policies.value & ASSET_PAIR_POLICIES.tradeableSecondaryMarket)) {
```

Comment:
you can plainly do `item.policies.value |= ASSET_PAIR_POLICIES.tradeableSecondaryMarket`